File: AtlasWeb/Core/Modules/Tibbi_Surec/Tetkik_Istem_Modulu/ProcedureResultInfoFormViewModel.ts
//$C2251E48
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { ProcedureResultInfo } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class ProcedureResultInfoFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.ProcedureResultInfoFormViewModel, Core';
    @Type(() => ProcedureResultInfo)
    public _ProcedureResultInfo: ProcedureResultInfo = new ProcedureResultInfo();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Tetkik_Istem_Modulu/ProcedureResultInfoFormViewModel.cs
//$CC92F591
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class ProcedureResultInfoServiceController
    {
    }
}

namespace Core.Models
{
    public partial class ProcedureResultInfoFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var procedureResultInfoFormService = new ProcedureResultInfoServiceController();
            procedureResultInfoFormService.ProcedureResultInfoFormInternal(this, objectContext);
            return (ProcedureResultInfo)objectContext.GetObject(this._ProcedureResultInfo.ObjectID, "ProcedureResultInfo");
        }
    }
}
